### DISCUSSION TITLE
Quick Question about Abfall.IO / AbfallPlus.de

### DISCUSSION THREAD

**Discussion by ferdiflix** [mampfes/hacs_waste_collection_schedule#631] [Q&A]
Hi there,
I would like to insert the abfall.io integration. My hometown Freiburg is listed there but not in the python script. So I want to make the manual workaround. I opend the [Website ](https://www.abfallwirtschaft-freiburg.de/de/private_haushalte/abfuhrtermine.php)and exported the ICS while opend the Network Tab. But in here I have differnet API calls than in the [example ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/abfall_io.md).

There I have 

- idhousenumber: 1025
- wastetypes: 1802,1146,1554

instead of 

- f_id_kommune: 2999
- f_id_strasse: 1087

Is there any known workaround for this?
Because if I enter this there will an error occure.

Thanks
Ferdi

**Comment by dldc** [comment]
Hi fellow Freiburgers, as it's only using GET parameters, I was able to use it with the simple ICS module, like this (yaml):

waste_collection_schedule:
   sources:
     - name: ics
       args:
         url: "https://api.abfall.io/?key=ba5c0a03ba41d81479797313161ced08&mode=export&idhousenumber=987&wastetypes=1146,627,17,1554,1802,1553&timeperiod={Y}0101-{Y}1231&type=ics"

Thanks for the nice addon, hope it helps!

Have a nice day,
Damiano
